rules/cross-platform/execution_pentest_eggshell_remote_admin_tool.toml

```diff
@@ -0,0 +1,27 @@
+[metadata]
+creation_date = "2021/01/12"
+maturity = "production"
+updated_date = "2021/01/12"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of EggShell Backdoor, EggShell is a known post exploitation pentesting tool for MacOS and
+Linux.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "EgShell Backdoor Execution"
+references = ["https://github.com/neoneggplant/EggShell"]
+risk_score = 73
+rule_id = "41824afb-d68c-4d0e-bfee-474dac1fa56e"
+severity = "high"
+tags = ["Elastic", "Host", "Linux", "macOS", "Threat Detection", "Execution"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and process.name:espl and process.args:eyJkZWJ1ZyI6*
+'''
+
```

Comment:
Since the tool is based on Python we could add https://attack.mitre.org/techniques/T1059/006/